# HG changeset patch
# User Sean Farley <sean.michael.farley@gmail.com>
# Date 1397847327 18000
#      Fri Apr 18 13:55:27 2014 -0500
# Node ID 7a1883c3f9ed3e8c9b9f97de7ecb5ed7e37c058a
# Parent  a47e3a6b57a431073b3e3244893e887f3aa9b1f5
make: use variable for python

diff --git a/admin/Makefile.python b/admin/Makefile.python
--- a/admin/Makefile.python
+++ b/admin/Makefile.python
@@ -39,11 +39,11 @@ include $(gagadmdir)/Makefile.build
 # Specific targets and rules
 
 ifneq ($(MOD_IDENTITY),)
 
 $(MOD_NAME): $(MOD_SOURCES) $(MOD_INCLUDES) $(MOD_EXTRAS) $(builddir)/setup.py
-	$(GAG_PYTHON_DISTFLAGS) python $(builddir)/setup.py build --build-base $(builddir) --build-lib $(builddir)
+	$(GAG_PYTHON_DISTFLAGS) @PYTHON@ $(builddir)/setup.py build --build-base $(builddir) --build-lib $(builddir)
 	$(MV) $(builddir)/*.$(SO_EXT) $(pythondir)
 
 $(builddir)/setup.py: setup.py.src Makefile
 	sed \
 	   -e 's?\$$MOD_IDENTITY?$(MOD_IDENTITY)?g' \
